test/Microsoft.AspNetCore.Authentication.WsFederation.Test/Microsoft.AspNetCore.Authentication.WsFederation.Test.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Today we only have a single test project, do you want to merge this into a new WsFed folder similar to OIDC?

Reply:
We've been trying to separate out OIDC into it's own project for a while but the work keeps getting backlogged. I'm not sure what we'd gain by merging this one.

Reply:
I didn't think that was a goal for OIDC, just to refactor/clean up tests.  I don't really care one way or another really long term (between one test project per package or one uber test project), but today since there's only one uber test project for all of auth, it seems weird to have this be stand alone.

Reply:
https://github.com/aspnet/Security/issues/349

Reply:
Personally I think having all the tests in one project are useful (when doing refactoring especially), running them all together doesn't even take long right now.  If these ever become slow then splitting them apart would be more useful, but if we are putting the functional tests in other repos like AuthSamples/Identity/templates, I don't think there's much value in splitting them up